### DISCUSSION TITLE
Map not rendering in Places

### DISCUSSION THREAD

**Discussion by MelAnubis** [photoprism/photoprism#3424] [Q&A] [answered]
The map in the Places view isn't working It shows markers where the photo locations are, but never loads the background. Ive reset the browser and tried both Edge and Chrome
Running Build [230506-9de9a3540] under Docker

**Comment by lastzero** [comment]
If you are not a member, it could be that this is because the level of detail isn't very good when using the built-in offline map. Unfortunately, you didn't share any specific coordinates, so it's impossible to reproduce your problem. For more detailed free maps, see the following issue:
- #2998

**Reply by MelAnubis** [reply]
Thanks !! seems it is a different issue. My map is totally empty regardless location in the world: 
![image](https://github.com/photoprism/photoprism/assets/134291207/b44cb0a5-1af6-4181-b846-f4372c4eda4a)

**Reply by lastzero** [reply]
These offline maps are pretty empty as it's just the shape of the continents plus the name of states and countries. I once did this on request on top of all the other work while I was on vacation (which is the only reason I had "time" to do this). With the help of contributors, we will soon provide better free maps as mentioned above. Thank you very much for your support and patience! ❤️

**Comment by alyssaholland99** [comment]
Yeah, I have the exact same issue.

**Reply by alyssaholland99** [reply]
They do appear to be the exact shape of the offline maps even though it's not set to offline in settings.

**Reply by lastzero** [reply]
Unfortunately, if you have not signed up for a membership, we can currently only provide you with the free offline maps - even if you have chosen a different map style. Please see the information and the issue linked above for how we will improve this. Note that even "free" maps require development time and server resources on our end. For this reason, we rely on your support and can only work as fast as our budget allows.

**Reply by alyssaholland99** [reply]
Hi; I was mainly speaking about the 'streets' map that used to be supported for non-members. Is this no longer the case?

**Reply by lastzero** [reply]
It was available for free, while we got it for free from Maptiler. However, it was listed as a member feature from the beginning. We now have too many users and PhotoPrism is no longer a small project. This means we have to pay for the maps as well as a long list of other infrastructure, including the reverse geocoding API, which all our users can still use for free.

**Reply by lastzero** [reply]
If you don't want to wait until we provide our own high-resolution maps (which we can then offer for free, like the Geocoding API), the easiest way to get better maps is to sign up for Essentials. It's just 2 EUR per month. If you then feel like it's not worth it, just cancel.

**Comment by lastzero** [comment, marked as answer]
Our new high-resolution world map is now available for testing at <https://demo.photoprism.app/library/places>. It is going to replace the currently included offline map **as the default option** and will be free forever.

Please see the following documentation for how to update your instance to use our preview image:
https://docs.photoprism.app/getting-started/updates/#development-preview
